schemas/README.md

```diff
@@ -992,6 +1455,122 @@ Line number in the file
 </tr>
 </table>
 
+## ModelPrimitive
+
+A primitive representing a 3D model file loaded from an external URL
+
+<table>
+  <tr>
+    <th>field</th>
+    <th>type</th>
+    <th>description</th>
+  </tr>
+<tr>
+<td><code>pose</code></td>
+<td>
+
+[Pose](#pose)
+
+</td>
+<td>
+
+Origin of model relative to reference frame
+
+</td>
+</tr>
+<tr>
+<td><code>scale</code></td>
+<td>
+
+[Vector3](#vector3)
+
+</td>
+<td>
+
+Scale factor to apply to the model along each axis
+
+</td>
+</tr>
+<tr>
+<td><code>color</code></td>
+<td>
+
+[Color](#color)
+
+</td>
+<td>
+
+Solid color to use for the whole model. If `use_embedded_materials` is true, this color is blended on top of the embedded material color.
+
+</td>
+</tr>
+<tr>
+<td><code>use_embedded_materials</code></td>
+<td>
+
+boolean
+
+</td>
+<td>
+
+Whether to use materials embedded in the model, or only the `color`
+
+</td>
+</tr>
+<tr>
+<td><code>url</code></td>
+<td>
+
+string
+
+</td>
+<td>
+
+URL pointing to model file. Either `url` or `mime_type` and `data` should be provided.
+
+</td>
+</tr>
+<tr>
+<td><code>mime_type</code></td>
+<td>
+
+string
+
+</td>
+<td>
+
+MIME type of embedded model (e.g. `model/gltf-binary`). Either `url` or `mime_type` and `data` should be provided.
```

Comment:
We should document: If I set `mime_type` with `url`, would it override the http response `content-type` header? Or will we always ignore this field with `url`?

The former option might be useful in certain situations (if you can't control the http server).

Reply:
As currently written it was intended to be that it's ignored if `url` is set. But I suppose we could allow it to be treated as an override.

Reply:
I think always treating it as an override (if set) probably makes more sense. I can think of why it might be useful, and it seems to simplify the logic. People can always leave it empty.